media-source-respec.html

```diff
@@ -705,7 +705,8 @@
           <h4>Seeking</h4>
           <p>Run the following steps as part of the "<i>Wait until the user agent has established whether or not the media data for the new playback position is available, and, if it is, until it has decoded enough data to play back that position"</i> step of the <a def-id="hme-seek-algorithm"></a>:</p>
           <ol>
-            <li>The media element looks for <a def-id="media-segments"></a> containing the <var>new playback position</var> in each <a>SourceBuffer</a> object in <a def-id="activeSourceBuffers"></a>.
+            <li>The media element looks for <a def-id="media-segments"></a> containing the <var>new playback position</var> in each <a>SourceBuffer</a> object in <a def-id="activeSourceBuffers"></a>. Any position within a <a def-id="timerange"></a> in the current value of the <a def-id="hme-buffered"></a> attribute MUST have a buffered media segment for that position.
```

Comment:
This doesn't read like a step.  I believe your intent is to add a step somewhat like this:

`If the readyState attribute is "ended" and the new playback position is within a TimeRange currently in HTMLMediaElement.buffered, then the seek must complete even if one or more currently selected track buffers have end times beyond the coded frame presentation interval.`

I feel like the contents of the note that follows belong in the step itself.